c++/test/CMakeLists.txt

```diff
@@ -42,6 +42,7 @@ add_executable (orc-test
   TestReader.cc
   TestRleDecoder.cc
   TestRleEncoder.cc
+  TestRleVectorDecoder.cc
```

Comment:
ditto, only append this file when option is enabled.

Reply:
Done.